docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,45 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+Clone the main REANA repository:
+
+```console
+$ git clone https://github.com/reanahub/reana.git && cd reana
+```
+
+You can setup a REANA development as follows:
+```console
+$ GITHUB_USER=<YOUR-GITHUB-USERNAME> make setup clone prefetch
+```
+
+
+Deploy REANA in development mode with code reload and [debugging capabilities](debugging):
+
+```console
+$ CLUSTER_FLAGS=debug.enabled=true make build deploy
+```
+
+!!! Info
+    You can deploy REANA locally but not in development mode as follows:
```

Comment:
- I think the order should be ascending: first explain how to do it without developer mode and live code, and only then explain the developer mode and debugging